intelmq/bots/experts/uri/expert.py

```diff
@@ -0,0 +1,60 @@
+# SPDX-FileCopyrightText: 2022 gutsohnCERT
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# -*- coding: utf-8 -*-
+
+from rfc3986 import uri_reference, validators, exceptions
+from intelmq.lib.bot import ExpertBot
+
+
+class URIExpertBot(ExpertBot):
+    """Parse the URI"""
+    overwrite = False
+
+    def process(self):
+        event = self.receive_message()
+        uri_validator = validators.Validator().require_presence_of('scheme')
```

Comment:
```suggestion
    _uri_validator = None
    
    def init(self):
        if rfc3986 is None:
            raise MissingDependencyError("rfc3986", version=">=1.5")
        self._uri_validator = validators.Validator().require_presence_of('scheme')
    
    def process(self):
        event = self.receive_message()
```

No need to re-initialize the validator everytime an event gets pulled